docs/user/dql/functions.rst

```diff
@@ -1770,9 +1772,10 @@ Description
 >>>>>>>>>>>
 
 Usage: month(date) returns the month for date, in the range 1 to 12 for January to December. The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid.
-The function month_of_year is also provided as an alias
+If an argument of type `TIME` is given, the function will use the current date to calculate an output.
```

Comment:
```suggestion
If an argument of type `TIME` is given, the function will use the current date.
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/199/commits/eac4b4e2ca1fb8a3041a572b9915a9be212981e4